src/app/public/src/addin-client.service.ts

```diff
@@ -31,6 +31,11 @@ export class AddinClientService {
    */
   public buttonClick: EventEmitter<any> = new EventEmitter(true);
 
+  /**
+   * Event emitted for button add-ins indicating that the button was clicked.
```

Comment:
Change function doc to `Event emitted for add-ins when context information has been updated.`

Reply:
fixed